### ISSUE TITLE
Does it support Android Neural Networks API?

### ISSUE THREAD

**Issue by manchengfenxu** [tensorflow/tensorflow#22209]
Have I written custom code N/A
OS Platform and Distribution Ubuntu 18.04
TensorFlow installed from anaconda
TensorFlow version 1.3.0
Bazel version N/A
CUDA/cuDNN version N/A
GPU model and memory N/A
Exact command to reproduce N/A
Mobile devic Pixel 2

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by harshini-gadige** [comment]
@manchengfenxu Hi, Could you please describe your problem/question clearly ?

**Comment by manchengfenxu** [comment]
@harshini-gadige Hi, I am asking if Android Neural Networks [API](https://developer.android.com/ndk/guides/neuralnetworks/)
can be deployed  to accelerate [TensorFlow Android Camera Demo ](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/examples/android)  on an Android device (Pixel 2)?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @achowdhery: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by harshini-gadige** [comment]
@achowdhery Hi, could you please look into this ?

**Comment by tensorflowbutler** [comment]
Nagging Assignee @achowdhery: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @achowdhery: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @achowdhery: It has been 44 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by jdduke** [comment]
All of our Android examples @ https://github.com/tensorflow/examples/tree/master/lite now support execution with [NNAPI](https://github.com/tensorflow/tensorflow/tree/master/tensorflow/lite/nnapi).
